doc/sphinx-guides/source/installation/config.rst

```diff
@@ -226,6 +229,39 @@ You also have the option to set a custom container name separator. It is initial
 
 ``./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.swift-folder-path-separator=-"``
 
+Amazon S3 Storage
++++++++++++++++++
+
+For institutions and organizations looking to use Amazon's S3 cloud storage for their installation, we can do can this with some brief setup steps. 
+
+You'll need an AWS account with an associated S3 bucket for your installation to use. From the S3 management console, you can poke around and get familiar with your bucket. Make note of the bucket's name and the region its data is hosted in. It's recommended you install the CLI tool `pip <https://pip.pypa.io//en/latest/>`_ to install the `AWS command line interface <https://aws.amazon.com/cli/>`_ if you don't have it.
+
+First, we'll get our access keys set up. If you already have your access keys configured, skip this step. From the command line, run:
+
+``pip install awscli``
+
+``aws configure``
+
+You'll be prompted to enter your Access Key ID and secret key, which should be issued to your AWS account. The subsequent config steps after the access keys are up to you.
+
+Second, with your access to your bucket in place, we'll want to navigate to ``/glassfish4/glassfish/bin/`` and execute the following ``asadmin`` commands to set up the proper JVM options. Recall that out of the box, Dataverse is configured to use local file storage. You'll need to delete the existing storage driver before setting the new one.
+
+``./asadmin $ASADMIN_OPTS delete-jvm-options "\-Ddataverse.files.storage-driver-id=file"``
+
+``./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.storage-driver-id=s3"``
+
+Then, we'll need to identify which S3 bucket we're using. Replace ``your_bucket_name`` with, of course, your bucket:
+
+``./asadmin create-jvm-options "-Ddataverse.files.s3-bucket-name=your_bucket_name"``
+
+Lastly, go ahead and restart your glassfish server. With Dataverse deployed and the site online, you should be able to create datasets and see them in your S3 bucket. Within a bucket, the folder structure emulates the one found in local file storage. 
+
+If you're set up to run REST-Assured tests (see http://guides.dataverse.org/en/latest/developers/testing.html), and don't want to use the GUI, you can run the ``S3AccessIT`` test:
```

Comment:
I'll take care of that, my apologies. Too used to adding to the dev guide instead of the installation guide!